src/infra/main/template.yaml

```diff
@@ -146,7 +151,6 @@ Globals:
 Resources:
   ### Interventions Processor ###
   InterventionsProcessorFunction:
-    DependsOn: InterventionsProcessorFunctionPolicy
     Type: AWS::Serverless::Function
```

Comment:
out of curiosity: was this updated template deployed successfully? i think this DependsOn was here because of a race condition